tests/integration/kleros.test.js

```diff
@@ -31,4 +35,35 @@ describe('Kleros', () => {
       mockArbitrable
     )
   })
+  it('web3wrapper', () => {
+    // Utils functions
+    expect(
+      klerosInstance.web3Wrapper._getNetworkIdIfExistsAsync()
+    ).resolves.toBeTruthy()
+    expect(klerosInstance.web3Wrapper.isAddress(randomEthAddress)).toBe(true)
+    expect(klerosInstance.web3Wrapper.getAccount(0)).toBeTruthy()
+    expect(klerosInstance.web3Wrapper.getProvider()).toBeTruthy()
+    expect(klerosInstance.web3Wrapper.getCoinbase()).toBeTruthy()
+    expect(klerosInstance.web3Wrapper.toWei('1', 'finney')).toBe(
+      1000000000000000
+    )
+    expect(
+      klerosInstance.web3Wrapper.fromWei('21000000000000', 'finney').toString()
+    ).toBe('0.021')
+    expect(
+      klerosInstance.web3Wrapper
+        .toBigNumber('200000000000000000000001')
+        .toString(10)
+    ).toBe('200000000000000000000001')
+    // Balance/user functions
+    expect(klerosInstance.web3Wrapper.getBalanceInWeiAsync()).toBeTruthy()
+    expect(
+      klerosInstance.web3Wrapper.doesContractExistAtAddressAsync(
+        randomEthAddress
+      )
+    ).resolves.toBeTruthy()
+    // Block tests
+    expect(klerosInstance.web3Wrapper.getBlock(10)).resolves.toBeTruthy()
```

Comment:
This one will be a race condition depending on what order the tests execute it. There might not be a block at height 10 if this test gets executed first

Reply:
I also agree in case of ganache. On normal blockchain should be the 10 block but I will change it  anyway.